teuthology/nuke.py

```diff
@@ -258,9 +258,9 @@ def remove_installed_packages(ctx):
     install_task.remove_packages(
         ctx,
         conf,
-        {"deb": install_task.deb_packages['ceph'] +
+        {"deb": install_task.PACKAGES['ceph']['deb'] +
          ['salt-common', 'salt-minion', 'calamari-server'],
```

Comment:
I agree it's weird, but I don't think we want to install them whenever we use the install task. Not trying to fix that problem here.